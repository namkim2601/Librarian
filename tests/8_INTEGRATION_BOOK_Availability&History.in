LiSt AvAiLaBlE
add book collection.csv 111111
list available
list available LoNg
BoOk HiStOry nosuchbook
book history 111111
add member member1
add member member2
rent 100001 111111
list available
list available long
relinquish 100001 111111
rent 100000 111111
relinquish 100000 111111
book history 111111
exit

