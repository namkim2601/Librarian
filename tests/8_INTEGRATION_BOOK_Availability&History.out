user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: To Kill a Mockingbird (Harper Lee)

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: No such book in system.

user: No rental history.

user: Success.

user: Success.

user: Success.

user: No books available.

user: No books available.

user: Success.

user: Success.

user: Success.

user: 100001
100000

user: Ending Library process.
